frontend/services/entities/mdm_apple.ts

```diff
@@ -69,12 +76,23 @@ export default {
     return sendRequest("GET", path);
   },
 
-  addVppApp: (teamId: number, appStoreId: string, platform: ApplePlatform) => {
+  addVppApp: (
+    teamId: number,
+    appStoreId: string,
+    platform: ApplePlatform,
+    isSelfService: boolean
+  ) => {
     const { MDM_APPLE_VPP_APPS } = endpoints;
-    return sendRequest("POST", MDM_APPLE_VPP_APPS, {
+    const postBody: IAddVppAppPostBody = {
       app_store_id: appStoreId,
       team_id: teamId,
       platform,
-    });
+    };
+
+    if (isSelfService) {
+      postBody.self_service = isSelfService;
```

Comment:
Out of curiosity, how does the API response if we were to send `self_service: false` explicitly?

Reply:
good question. it's not specified in the requirement docs. in this case it will always be `true` here
but I think its still worth it to have as an argument that can be passed in.